### DISCUSSION TITLE
expand option not working

### DISCUSSION THREAD

**Discussion by actuallySaptarshi** [pocketbase/pocketbase#5465] [Q&A] [answered]
I'm trying to build a realtime chat app using react and pocketbase as backend. Most of the things are working completely fine but there is an issue with the 
{ expand: 'user' }
options specifically 
`const resultList = await pb.collection('messages').getList(1, 50, {sort: 'created', expand: 'user'})`

and 

`pb.collection('messages').subscribe('*', async ({ action, record}) => {
      if (action === 'create') {
        messagesUpdated([...messages, record])
    }
    }, { expand: 'user' })`

Everything else like updating messages in realtime is working completely fine but the expand option seems to just not work.
Here are a few more screenshots which might help:
![Chat App 2](https://github.com/user-attachments/assets/476c90fe-cb36-4407-aa76-fe988d4aea0e)
![Chat App](https://github.com/user-attachments/assets/5f1da9b3-ace4-40f0-a142-bc45961bd3c5)
![server](https://github.com/user-attachments/assets/85acd3ef-fa9e-43f4-b74a-3baf7294baa7)

**Comment by ganigeorgiev** [comment]
There are not enough details to reproduce your issue but I don't think it is PocketBase related.

It is working correctly for me in the live demo and we have several integration tests that confirms the correct behavior.

Note that the expanded user will be available under the `expand` object (the json screenshot doesn't show the complete output so I can't determine whether that is the case here).

Make sure that you are also using the latest versions of the PocketBase executable and JS SDK.

If you are still not able to identify the culprit, feel free to export you collections configuration as json (_Admin UI > Settings > Export collections_), provide a more detailed code sample how you are sending the request and I'll try to investigate it, but as of now I don't think there is much I or anyone else help here.

**Reply by actuallySaptarshi** [reply]
[pb_schema.json](https://github.com/user-attachments/files/16832403/pb_schema.json)

Hope this helps,
Also I am using the latest version of pocketbase i.e. v0.22.20 with the latest version of js as well

**Reply by ganigeorgiev** [reply, marked as answer]
@actuallySaptarshi Based on the provided collection your relation field is named "field" not "user", aka. either rename your collection field from "field" -> "user" or replace your SDK call with `expand: "field"`.

**Reply by actuallySaptarshi** [reply]
That solved my issue! Thanks a lot :)
